modules/client-configuration/pages/contact-methods-pushssh.adoc

```diff
@@ -0,0 +1,101 @@
+[[contact-methods-pushssh]]
+= Push via Salt SSH (with tunnel)
+
+
+Push via SSH (with tunnel) is used in environments where clients cannot reach the {productname} Server directly.
+In this environment, clients are located in a firewall-protected zone called a DMZ.
+No system within the DMZ is authorized to open a connection to the internal network, including the {productname} Server.
+
+This SSH method creates an encrypted tunnel from the {productname} Server on the internal network to the clients located on the DMZ.
+After all actions and events are executed, the tunnel is closed.
+
+The server uses SSH to contact the clients at regular intervals, checking in and performing scheduled actions and events.
+
+
+
+[IMPORTANT]
+====
+Re-installing systems using the provisioning model is not currently supported on clients managed with Push via SSH.
+====
+
```

Comment:
I think it would be nice to add one more `IMPORTANT` note as some of the users are still trying to do it. As the tunnel is used to provide the sccess to the server throug the encripted tunned, the repos assigned to the `salt-ssh` client with tunnel are provided through this tunnel only, so it's not possible to use package manager tools directly from the system as the repos available only while the tunnel is up (basically speaking if the session is iniciated by the server), so it means all package managing operation on the client can be performed from the server side only.

Reply:
ok, I tried to add such an admonition, re-using your phrases.  I hope I got it right.